### PR TITLE
Upgrading dependencies

### PR DESCRIPTION
**Pull request by joeldenning**
I did this by running `yarn upgrade-interactive --latest`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,48 +20,48 @@
...
+    "webpack-dev-server": "^3.9.0"
   },
   "dependencies": {
     "busboy": "^0.3.1",
     "classnames": "^2.2.6",
     "ejs": "^2.7.1",
     "email-validator": "^2.0.4",
     "express": "^4.17.1",
-    "firebase": "^6.4.2",
+    "firebase": "^7.2.2",
```
Comment (joeldenning) [inline package.json]: See https://firebase.google.com/support/release-notes/js#version_700_-_september_26_2019 - the breaking change does not appear to impact us.

File: backend/server.js [hunk 2]
```diff
@@ -11,8 +11,6 @@ if (process.env.NODE_ENV !== "production") {
   app.use(morgan("tiny"));
 }
 
-console.log("dist exists", fs.existsSync(path.join(__dirname, "../dist")));
```
Comment (joeldenning) [inline backend/server.js]: This shouldn't have been here in the first place.

File: package.json [hunk 3]
```diff
@@ -20,48 +20,48 @@
...
+    "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-react": "^7.6.3",
     "babel-loader": "^8.0.6",
     "clean-webpack-plugin": "^3.0.0",
-    "concurrently": "^4.1.2",
+    "concurrently": "^5.0.0",
```
Comment (joeldenning) [inline package.json]: See https://github.com/kimmobrunfeldt/concurrently/releases/tag/v5.0.0 - the breaking change doesn't impact us.

File: package.json [hunk 4]
```diff
@@ -20,48 +20,48 @@
...
     "prop-types": "^15.7.2",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-icons": "^3.7.0",
-    "react-router": "^5.0.1",
-    "react-router-dom": "^5.0.1"
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
+    "react-icons": "^3.8.0",
+    "react-router": "^5.1.2",
```
Comment (scotttball) [inline package.json]: I'm glad for this. 5.1.x added hooks for useHistory, etc.

**Pull request merged**